app/models/station.rb

```diff
@@ -33,4 +33,14 @@ def self.newest
   def self.oldest
     find_by(installation_date: minimum(:installation_date))
   end
+
+  def most_frequent_destination
+    station_id = start_trips.group(:end_station_id).order('count(id) DESC').count(:id).keys.first
```

Comment:
@AliSchlereth @iandouglas This feels like a really gross way of achieving this result... I've been searching for a way to go from Station -> list of trips -> most common station in a single active record query, but this is just what i did to make it work. 

Is there something I should be exploring to do this? I've not been able to find a join query that worked so far.

Reply:
I agree that what you've built is not ideal as it hits the DB once to get the station_id and a second time to find the appropriate station. My recommendation is to play with using select. Check out the last section of this lesson https://github.com/turingschool/backend-curriculum-site/blob/gh-pages/module2/lessons/joins_group_order.md for a bit more guidance. I believe if you put in the select your above query will return back to you the object itself rather than the hash with the key of the id.